FAQ.md

```diff
@@ -0,0 +1,17 @@
+# FAQ
+Frequently Asked Questions
+
+## Where's my ticket to the hackathon?
+We had an extensive *application* process that was detailed by Quincy on at https://hackathon.freecodecamp.org/ . If you *successfully* completed this process, and you were *one of the first 300 to do so* (for the in-person event), then you were sent a ticket to the email address provided in your application. If you do not have a ticket then I'm sorry, but you cannot participate.
+
+## What is the JAMstack?
+The JAMstack is a powerful new approach for deploying fast, highly scalable sites and applications that don't require backend infrastructure. Thanks to recent advancements in the web platform, developers can now run any web property, from simple sites to complex applications, on global CDNs and without a single web server. Learn more about how the JAMstack works [here](https://jamstackconf.com/what-is-jamstack).
```

Comment:
While this explanation worked well for the conference, I think it may confuse participants. Here's a suggestion:

> JAM stands for JavaScript, APIs, and Markup, joining forces in a powerful new approach for deploying fast, highly scalable sites and applications. It's made possible by separating frontend and backend — this frees the frontend to be served quickly in pre-rendered files over a global Content Delivery Network (CDN). Backend tasks like authentication, ecommerce, and database storage are handled by serverless cloud functions and APIs (provided by third-party services or custom-made). Learn more about how the JAMstack works [here](https://jamstackconf.com/what-is-jamstack).

Reply:
lol, oops. Merged while I typed. 😛